src/containers/Repo/Grader/GraderPage.js

```diff
@@ -19,7 +20,9 @@ import { OverlayTrigger, Tooltip } from 'react-bootstrap';
   state => ({
     repo: state.repo.repo,
     students: state.grade.students,
-    error: state.grade.error
+    error: state.grade.error,
+    submitting: state.grade.submitting,
+    submission: state.grade.submission
```

Comment:
Don't add to asyncConnect this is to help with loading the props when first opening the page but email submission is not related to that